views/mail/_partials/order/payment_state.htm

```diff
@@ -1,9 +1,9 @@
 name = "Payment state"
 ==
 {% if order.isPaid %}
-    The order has been paid on {{ order.payment_log.created_at }}.
+    {{ 'offline.mall::mail.order.partials.has_been_paid_on' | trans() }} {{ order.payment_log.created_at }}.
```

Comment:
Let's use a placholder here.

https://laravel.com/docs/5.5/localization#replacing-parameters-in-translation-strings

```suggestion
    {{ 'offline.mall::mail.order.partials.has_been_paid_on' | trans({date: order.payment_log.created_at}) }}.
```